testing/postfwd2/APKBUILD

```diff
@@ -0,0 +1,48 @@
+# Contributor: Simon Frankenberger <simon-alpine@fraho.eu>
+# Maintainer: Simon Frankenberger <simon-alpine@fraho.eu>
+pkgname=postfwd2
+_pkgname=${pkgname:0:-1}
+pkgver=1.39
+pkgrel=0
+pkgdesc="postfwd is written in perl to combine complex postfix restrictions in a ruleset similar to those of the most firewalls."
```

Comment:
The description should not repeat the package/project name. Maybe you can think of a more succinct description?